storm-core/src/jvm/org/apache/storm/task/ShellBolt.java

```diff
@@ -158,8 +158,8 @@ public void execute(Tuple input) {
 
             _pendingWrites.putBoltMsg(boltMsg);
         } catch(InterruptedException e) {
-            String processInfo = _process.getProcessInfoString() + _process.getProcessTerminationInfoString();
-            throw new RuntimeException("Error during multilang processing " + processInfo, e);
+            // It's likely that Bolt is shutting down so no need to throw RuntimeException
```

Comment:
My bad @HeartSaVioR. You can merge this for now. I have captured the improvement in 
https://issues.apache.org/jira/browse/STORM-1600